### PR TITLE
Use the new Fixer.io endpoint

### LINKED ISSUE #36010: Fixer IO API Endpoint changed
### Preconditions and environment

- Magento version: 2.4.5
- Anything else that would help a developer reproduce the bug

### Steps to reproduce

> Simply if you now go to configure currency exchange using Fixer IO they will direct you to a new platform with a new API Endpoint https://apilayer.com/marketplace/fixer-api
> Admin - Stores - Currency Rates - Select - Fixex.io as Import service and click on Import

### Expected result

It imports currency rates successfully

### Actual result

The import is failing due to the new API keys only compatible with the new API Endpoint which must be updated/corrected from within M2

### Additional information

_No response_

### Release note

_No response_

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [X] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

### PR DESCRIPTION
**Pull request by daniloargentiero**
### Description (*)
Fixer.io changes his endpoint, then Magento Fixer.io service stops to work. With this PR Magento use the new Fixer.io endpoint

### Manual testing scenarios (*)
Go to Stores > Currency Rates , select Fixer.io as service and click on "Import". An error will appear.

### Contribution checklist (*)
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [X] All new or changed code is covered with unit/integration tests (if applicable)
 - [X] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [X] All automated tests passed successfully (all builds are green)

Fixes https://github.com/magento/magento2/issues/36010

### CODE REVIEW

**Comment by daniloargentiero** [comment]
@magento run all tests

**Comment by daniloargentiero** [comment]
@magento run all tests

**Comment by daniloargentiero** [comment]
@magento run all tests

**Comment by daniloargentiero** [comment]
@magento run all tests

**Comment by daniloargentiero** [comment]
Magento's test asks me to fix the currencyconverterapi api too. I'm working on it and I will push a new commit with it.

File: app/code/Magento/Directory/Model/Currency/Import/FixerIo.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ class FixerIo extends AbstractImport
     /**
      * @var string
      */
-    public const CURRENCY_CONVERTER_URL = 'http://data.fixer.io/api/latest?access_key={{ACCESS_KEY}}'
+    private const CURRENCY_CONVERTER_URL = 'https://api.apilayer.com/fixer/latest?apikey={{ACCESS_KEY}}'
```
Comment (ihor-sviziev) [inline app/code/Magento/Directory/Model/Currency/Import/FixerIo.php]:
```suggestion
    public const CURRENCY_CONVERTER_URL = 'https://api.apilayer.com/fixer/latest?apikey={{ACCESS_KEY}}'
```

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by daniloargentiero** [comment]
@magento run all tests

**Comment by daniloargentiero** [comment]
@magento run all tests

**Comment by daniloargentiero** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
HI @daniloargentiero,
As it was described in https://github.com/magento/magento2/pull/36077#issuecomment-1236708857, this PR isn't solving the issue because it's not passing access key as header. As a result of this, I'm closing this PR in favor to https://github.com/magento/magento2/pull/36077

**Pull request closed without merging**